packages/token-introspection/src/openapi/token-introspection.yaml

```diff
@@ -0,0 +1,175 @@
+openapi: 3.1.0
+info:
+  title: Rafiki Authorization Server - Resource Server Connection
+  version: '1.0'
+  license:
+    name: Apache 2.0
+    identifier: Apache-2.0
+  summary: Rafiki Authorization Server - Resource Server Connection
+  description: 'The Open Payments API is secured via [GNAP](https://datatracker.ietf.org/doc/html/draft-ietf-gnap-core-protocol). This specification describes the connection between the Rafiki Open Payments Authorization Server and the Rafiki Open Payments Resource Server, which is an opinionated version of the [Grant Negotiation and Authorization Protocol Resource Server Connections](https://datatracker.ietf.org/doc/html/draft-ietf-gnap-resource-servers).'
+  contact:
+    email: tech@interledger.org
+servers:
+  - url: 'https://openpayments.guide/auth'
+tags:
+  - name: introspection
+    description: Token introspection
+paths:
+  /:
+    parameters: []
+    post:
+      summary: Introspect Access Token
+      operationId: post-introspect
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - properties:
+                      active:
+                        enum:
+                          - false
+                    required:
+                      - active
+                  - $ref: '#/components/schemas/token-info'
+              examples:
+                Token Introspection:
+                  value:
+                    active: true
+                    grant: 1ee48d97-8fa1-4ab4-b89d-95284b665517
+                    access:
+                      - type: outgoing-payment
+                        actions:
+                          - create
+                          - read
+                        identifier: 'https://openpayments.guide/alice'
+                        limits:
+                          interval: 'R12/2019-08-24T14:15:22Z/P1M'
+                          receiver: 'https://openpayments.guide/bob/incoming-payments/48884225-b393-4872-90de-1b737e2491c2'
+                          sendAmount:
+                            value: '500'
+                            assetCode: USD
+                            assetScale: 2
+                    key:
+                      proof: httpsig
+                      jwk:
+                        alg: EdDSA
+                        kid: 20f24ce2-a5f6-4f28-bf7d-ed52d0490187
+                        kty: OKP
+                        use: sig
+                        crv: Ed25519
+                        x: AAAAC3NzaC1lZDI1NTE5AAAAIK0wmN/Cr3JXqmLW7u+g9pTh+wyqDHpSQEIQczXkVx9q
+                    client_id: 581a1dd9-aef5-4b1a-9d78-c1afc4a5fdbf
```

Comment:
Not a big deal since it's an example, but would it be more accurate if this was a hash instead of a uuid?

Reply:
:+1: but `client_id` is living on borrowed time
- https://github.com/interledger/rafiki/issues/737